libnetwork/controller.go

```diff
@@ -870,9 +870,10 @@ func (c *Controller) NewSandbox(containerID string, options ...SandboxOption) (*
 	}
 	c.mu.Unlock()
 
-	sandboxID := stringid.GenerateRandomID()
-	if runtime.GOOS == "windows" {
-		sandboxID = containerID
+	// TODO(thaJeztah): given that a "containerID" must be unique in the list of sandboxes, is there any reason we're not using containerID as sandbox ID on non-Windows?
+	sandboxID := containerID
+	if runtime.GOOS != "windows" {
+		sandboxID = stringid.GenerateRandomID()
 	}
```

Comment:
Oh! I was planning to move this inside the `if sb == nil` branch below, because it's only used there; let me do that in this PR.

Reply:
done 👍